marketplace-service/src/main/java/com/axonivy/market/controller/ProductController.java

```diff
@@ -45,18 +52,33 @@ public class ProductController {
   private final PagedResourcesAssembler<Product> pagedResourcesAssembler;
 
   public ProductController(ProductService productService, GitHubService gitHubService, ProductModelAssembler assembler,
-      PagedResourcesAssembler<Product> pagedResourcesAssembler) {
+                           PagedResourcesAssembler<Product> pagedResourcesAssembler) {
     this.productService = productService;
     this.gitHubService = gitHubService;
     this.assembler = assembler;
     this.pagedResourcesAssembler = pagedResourcesAssembler;
   }
 
-  @Operation(summary = "Find all products", description = "Be default system will finds product by type as 'all'")
   @GetMapping()
-  public ResponseEntity<PagedModel<ProductModel>> findProducts(@RequestParam(name = TYPE) String type,
-      @RequestParam(required = false, name = KEYWORD) String keyword,
-      @RequestParam(name = LANGUAGE) String language, Pageable pageable) {
+  @Operation(summary = "Find all products", description = "Be default system will finds product by type as 'all'", parameters = {
```

Comment:
Please replace "Find all products" to "Retrieve a paginated list of all products, optionally filtered by type, keyword, and language."